sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -653,53 +653,29 @@ public void flush(long timeoutMillis) {
     return sentryId;
   }
 
-  @Override
-  public @NotNull ITransaction startTransaction(
-      final @NotNull TransactionContext transactionContext,
-      final @Nullable CustomSamplingContext customSamplingContext,
-      final boolean bindToScope) {
-    return createTransaction(
-        transactionContext, customSamplingContext, bindToScope, null, false, null, false, null);
-  }
-
   @ApiStatus.Internal
   @Override
   public @NotNull ITransaction startTransaction(
-      @NotNull TransactionContext transactionContext,
-      @Nullable CustomSamplingContext customSamplingContext,
-      boolean bindToScope,
-      @Nullable Date startTimestamp) {
+      final @NotNull TransactionContext transactionContext,
+      final @NotNull TransactionOptions transactionOptions) {
```

Comment:
I noticed that we now have `transactionOptions` param which is nice, my question is, for all the methods that have changed to accept this new param, are they all `@ApiStatus.Internal` or are we breaking Public APIs as well?

Reply:
I only changed the internal ones for now.